src/common/task.cc

```diff
@@ -145,23 +145,23 @@ void free_task_builder(TaskBuilder *builder) {
 }
 
 bool TaskID_equal(TaskID first_id, TaskID second_id) {
```

Comment:
The plan is for these `equal` and `is_nil` methods to go away (e.g., in a future PR) and just use `==` and `is_nil`, right?

Reply:
Yeah!